OrbitCore/TimerManager.h

```diff
@@ -35,9 +35,8 @@ class TimerManager {
   void StartClient();
   void StopClient();
 
-  void Add(const Timer& a_Timer);
-  void Add(const Message& a_Message);
-  void Add(const ContextSwitch& a_CS);
+  void AddTimer(const Timer& a_Timer);
```

Comment:
Was it necessary to rename? It seems this should be ok with just Add, no?

Reply:
Because the function was overloaded, but that's not necessary and I don't really like it in a case like this, it should be clear from the name what is being added, not just from the type of the argument.

Reply:
In general I am a friend of overloading, but in this case, I see that it was confusing. I would be fine with both.

Reply:
I would generally advise against renaming existing functions just because you don't like the way there are named unless there are really good reasons. And I think here it's mostly a matter of opinion. I prefer the overloaded style here, because it's more concise. You have more, unnecessary, verbosity now: The signature reads "AddTimer(...Timer..)" and all call sites are "AddTimer(timer)" and "AddContextSwitch(CS)".